### PR TITLE
add InstallationParams to Application

### PR DESCRIPTION
**Pull request by caneleex**

### CODE REVIEW

**Review by TopiSenpai — COMMENTED** [review]
I know tags are not used atm
But maybe you can add them too just for completeness

File: discord/application.go [hunk 1]
```diff
@@ -6,24 +6,26 @@ import (
...
+	Name                string              `json:"name"`
+	Icon                *string             `json:"icon,omitempty"`
+	Description         string              `json:"description"`
+	RPCOrigins          []string            `json:"rpc_origins"`
+	BotPublic           bool                `json:"bot_public"`
+	BotRequireCodeGrant bool                `json:"bot_require_code_grant"`
+	TermsOfServiceURL   *string             `json:"terms_of_service_url,omitempty"`
+	PrivacyPolicyURL    *string             `json:"privacy_policy_url,omitempty"`
+	InstallationParams  *InstallationParams `json:"install_params"`
+	Tags                *[]string           `json:"tags"`
```
Comment (TopiSenpai) [inline discord/application.go:19]:
```suggestion
	Tags                []string            `json:"tags"`
```
this should not be a pointer

Comment (caneleex) [reply discord/application.go:19]: the `tags` key is missing if there are no tags, it doesn't return an empty array, so it'd have to be nil, wouldn't it?

Comment (TopiSenpai) [reply discord/application.go:19]: Slices can be nil already so it's fine

**Review by TopiSenpai — APPROVED** [review]
lgtm

**Pull request merged**